cobaya/samplers/polychord/polychord.py

```diff
@@ -142,11 +154,9 @@ def initialize(self):
                    "cluster_posteriors", "write_resume", "read_resume",
                    "write_stats", "write_live", "write_dead", "write_prior",
                    "maximise", "compression_factor", "synchronous", "base_dir",
-                   "file_root", "seed", "grade_dims", "grade_frac", "nlives"]
+                   "file_root", "grade_dims", "grade_frac", "nlives"]
```

Comment:
Is there a reason seed has been removed?

Reply:
Does this mean this closes #3?

Reply:
yes